src/browser/api_protocol/api_handlers/window.ts

```diff
@@ -126,7 +126,7 @@ function stopFlashWindow(identity: Identity, message: APIMessage, ack: Acker): v
 
 function setWindowBounds(identity: Identity, message: APIMessage, ack: Acker): void {
     const { payload } = message;
-    const { top, left, width, height } = payload;
+    const { top, left, width, height } = payload.bounds;
```

Comment:
see comment in js adapter, leave these top level in the payload to ensure backwards compatibilty

Reply:
Ahhh duh, reverted.